test/Dispatcher.base.t.sol

```diff
@@ -189,4 +190,28 @@ contract Base is IbcEventsEmitter, ProofBase, TestUtilsTest {
             ? abi.encodePacked('{"result":"', Base64.encode(ack.data), '"}')
             : abi.encodePacked('{"error":"', ack.data, '"}');
     }
+
+    // Store connection in channelid to connection mapping using store
+    function _storeChannelidToConnectionMapping(bytes32 channelId, bytes32 connection) internal {
```

Comment:
nit: can we pass `string connectionId` so its consistent with API param types. Can convert to bytes32 programatically in this func

Reply:
I wanted to programmatically convert the connection from string into bytes32, but the problem is that in practice the way that solidity stores a string in memory in this array is not really reproducible (I think it might be some sort of optimization that solidity is doing) 

for example, the string "connection-0" should map to a storage slot as `0x636f6e6e656374696f6e2d300000000000000000000000000000000000000018`. When I try:

- `abi.encodePacked("connection-0")`, it produces `0x636f6e6e656374696f6e2d300000000000000000000000000000000000000000` - prefix is correct but missing the 18 at the end
-`abi.encode("connection-0")` produces  `0x636f6e6e656374696f6e2d300000000000000000000000000000000000000000` and `0x0000000000000000000000000000000000000000000000000000000000000020` which fits into two seperate storage slots.